DEHPEcosimPro/Views/MainWindow.xaml

```diff
@@ -84,11 +85,28 @@
                                         <RowDefinition Height="*" />
                                     </Grid.RowDefinitions>
                                     <common:HubBrowserHeader DataContext="{Binding HubDataSourceViewModel.HubBrowserHeader}" Grid.Row="1" />
-                                    <objectBrowser:ObjectBrowser Grid.Row="2">
-                                        <objectBrowser:ObjectBrowser.DataContext>
-                                            <Binding Path="HubDataSourceViewModel.ObjectBrowser"></Binding>
-                                        </objectBrowser:ObjectBrowser.DataContext>
-                                    </objectBrowser:ObjectBrowser>
+                                    <TabControl Grid.Row="2">
+                                        <TabItem>
+                                            <TabItem.Header>
+                                                <TextBlock Text="Element Definitions" />
```

Comment:
Should be "Engineering Model" as in the future it will not just be ElementDefinitions.